ui/src/wizard/AzureWizardUserInput.ts

```diff
@@ -177,8 +174,14 @@ export class AzureWizardUserInput implements IWizardUserInput {
                     }
                 }
             }
-            return <TPick[]>picks;
+        } else {
+            picks = picks.concat(...groups.map(g => g.picks));
```

Comment:
Can't we directly return here without `else`?  Also I think we can do a picks.push() and return that.